src/components/forms/Combobox.tsx

```diff
@@ -284,6 +287,7 @@ interface EntryOptions<T = any> {
   label: string;
   value: string;
   labelFn?: (resource: T) => string | JSX.Element;
+  displayValueFn?: (resource?: T) => string;
```

Comment:
I got the idea now, but the naming is wrong.

Can we rename `labelFn` to maybe `dropdownLabelFn` and `displayValueFn` to `inputLabelFn`? This way the situation is a bit more clear.

Reply:
@beganovich Renamed!